enaml/qt/q_popup_view.py

```diff
@@ -21,6 +21,9 @@
 from .q_single_widget_layout import QSingleWidgetLayout
 
 
+IS_X11 = QApplication.platformName() == "xcb"
```

Comment:
This feels like something that could be useful in other places and that we are likely to forget if it is here. I would prefer to have this in the init of the qt package.

Reply:
platformName returns a blank string when used in `__init__.py`, I guess it is only set after the application is created.